include/seqan/system/file_async.h

```diff
@@ -813,20 +813,17 @@ class File<Async<TSpec> >:
 struct AiocbWrapper :
     public aiocb
 {
-    AiocbWrapper()
-    {}
+    AiocbWrapper() = default;
+    AiocbWrapper(AiocbWrapper &&) = default;
 
     AiocbWrapper(AiocbWrapper & other, Move) :
```

Comment:
Can you check if this move hack is still required? Now, that we actually have move constructors?
If you want to, you can also open a new ticket for this.